test/scalarstats.jl

```diff
@@ -23,8 +23,8 @@ using Base.Test
 @test_approx_eq genmean([1,1,2,3], 0) (6.0)^(1/4)
 @test_approx_eq genmean([1.2,-0.5,0], 2) sqrt(169/300)
 @test_approx_eq genmean([16/9,0.25,1.0], 1.5) (755/648)^(2/3)
-@test_approx_eq_eps genmean([1,1,2,3], -1e-8) (6.0)^(1/4) 1e-8
-@test_approx_eq_eps genmean([0.98,1.02], 1e4) 1.02 1e-4
+@test_approx_eq_eps genmean([1,1,2,3], -1e-8) (6.0)^(1/4) 1e-8# Test numerical stability for `p` close to 0 (genmean should be close to geometric mean).
```

Comment:
Lines should be under 92 characters, so please put the comments on separate lines (also, the common convention is to add a space before `#` when on the same line).